patches/README.md

```diff
@@ -0,0 +1,6 @@
+# rM Hacks / Patches
+
+All the files in the current directory are marked as "All Rights Reserved,"
+which means that they are protected by copyright, and no specific license
+information is available for them.  Users are not allowed to modify or
```

Comment:
"Don't repeat yourself". Especially with license/contact terms it is important to have a single source of truth, the license file. I recommend likng it like in the root README. 

AFAIK stating a number of prohibitions (modification and distribution) does not mean that anything else (usage) is fair game, especially with "all rights reserved".
Hence maybe something like this: "Users are allowed to install and use these files via the installation script provided."

Reply:
Sorry for being pedantic about DRY. 😅  I've been bitten by that one too many times..

That paragraph sounds good to me

Reply:
Don't be sorry, I really appreciate your inputs, thanks!